### DISCUSSION TITLE
How to review

### DISCUSSION THREAD

**Discussion by tbilsbor** [st3v3nmw/obsidian-spaced-repetition#692] [Q&A]
Seems like I must be missing something obvious here but I can't find the option to review cards. There's the sidebar button that says "Review flashcards" which seems like it would be the place but it just shows me the decks I've created. There's no option to actually do the review. I've tried double-clicking on the decks as well. Tried restarting Obsidian, reinstalling the plugin, and looking through the documentation, but no luck.

**Comment by AB1908** [comment]
Did you try the command palette?

**Comment by tbilsbor** [comment]
Yeah, and there's the option to review and it brings up the list of decks but I can't get anywhere from there. I think something might be f'd up with my Obsidian installation, because UX-wise it definitely seems like something should be happening that isn't happening.

**Reply by AB1908** [reply]
Can you share screenshots? Have you written any flashcards in any of the formats?

**Comment by tbilsbor** [comment]
Reinstalled Obsidian and it's working now. Confirmed that the thing I was intuitively expecting it to do (open a deck for review when clicked on) was not happening.
